index.html

```diff
@@ -1,32 +1,22 @@
 <!doctype html>
 <html>
-<head>
+  <head>
     <title>We'll be right back!</title>
-    <style type="text/css">
-        body {
-            font-family: arial, sans-serif;
-            font-size: 16pt;
-        }
-        #content {
-            width: 960px;
-            margin: 5em auto;
-        }
-        #logo {
-            margin-top: 10em;
-        }
-    </style>
-</head>
-<body>
-<div id="content">
-    <h1>We&rsquo;ll be right back!</h1>
-
-
-    <p>This web site is unavailable at the moment because we&rsquo;re doing
-    some essential maintenance. Sorry for the inconvenience!</p>
-
-    <p>Please check back soon.</p>
-
-    <p id="logo"><a href="https://www.zooniverse.org/"><img src="logo.png" alt="Zooniverse"></a></p>
-</div>
-</body>
+    <link rel="stylesheet" href="./main.css">
+    <link href="https://fonts.googleapis.com/css?family=Karla" rel="stylesheet">
+  </head>
+  <body>
+    <img src="img/shape.svg">
```

Comment:
Set this as a background image in the CSS instead. Check out the `background-image` and `background-repeat` properties.